CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing
+
+## Setup your development environment
+
+1) Install [pyenv](https://github.com/pyenv/pyenv) for your system to use the correct version of python specified in `.python-version`. Make sure your $PATH now includes `~/.pyenv/bin` && `~/.pyenv/shims`.
+
+2) Install [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) to isolate project dependencies which keeps `requirements.txt` clean and frozen. 
+
+3) Install the version of python listed in `.python-version`:
+
+   $ pyenv install <paste only the full version number, e.g. 3.10.4>
+
+4) Create the virtual env:
+
+   $ pyenv virtualenv <paste only the full version number, e.g. 3.10.4> postgresml
+
+5) Install the dependencies:
+
+    $ pip install -r requirements.txt
+
+6) If you ever add new dependencies, freeze them:
+
+    $ pip freeze > requirements.txt
+
+7) Make sure requirements.txt has no changes, which indicates your virtual environment is setup correctly.
+
+    $ git status
+
+8) Create the development databases in the postgres cluster running inside docker:
+
+    $ psql -c "CREATE DATABASE pgml_development" postgres://postgres:postgres@127.0.0.1:5433/

```

Comment:
We can do this in `bin/entrypoint.sh`

Reply:
Cool. I think we also need to setup a script for non docker environments. I like developing outside of docker for iteration speed. Maybe we can have a single setup script that works for both.

Reply:
Yeah for sure, local dev without docker is much better!